stackdriver/client_test.go

```diff
@@ -120,8 +120,20 @@ func TestEmptyRequest(t *testing.T) {
 }
 
 func TestResolver(t *testing.T) {
-	addressesToTest := []string{"stackdriver.invalid", "2001:db8::"}
-	for _, address := range addressesToTest {
+	tests := []struct {
```

Comment:
I think this is similar, but the intention is more explicit and it avoids corner cases in the test code below:
	tests := []struct {
		urlHostname       string
		expectedAddress string
	}{
		{
			"stackdriver.invalid",
			"stackdriver.invalid",
		},
		{
			"[2001:db8::]",
			"2001:db8::",
		},
	}

Reply:
Updated. I use the term `host` instead of `hostname` to match the definition in RFC2732.